src/asset-relocator.js

```diff
@@ -798,7 +801,13 @@ module.exports = async function (content, map) {
                 replacement = '__non_webpack_require__(' + replacement + ')';
               return replacement;
             case 'directory':
-              return emitAssetDirectory(path.resolve(value));
+              // do not emit asset directories lower than the package base itself
```

Comment:
I might have my terminology wrong but I would use the opposite.

Something like

> do not emit asset directories above the package base itself

Reply:
Like you move "up" the tree to a parent directory and "down" the tree to a leaf directory/file.